bigquery_etl/backfill/parse.py

```diff
@@ -152,21 +152,22 @@ def entries_from_file(cls, file: Path) -> List["Backfill"]:
                 backfills = yaml.load(yaml_stream, Loader=UniqueKeyLoader) or {}
 
                 for entry_date, entry in backfills.items():
-                    excluded_dates = []
-                    if "excluded_dates" in entry:
-                        excluded_dates = entry["excluded_dates"]
-
-                    backfill = cls(
-                        entry_date=entry_date,
-                        start_date=entry["start_date"],
-                        end_date=entry["end_date"],
-                        excluded_dates=excluded_dates,
-                        reason=entry["reason"],
-                        watchers=entry["watchers"],
-                        status=BackfillStatus[entry["status"].upper()],
-                    )
-
-                    backfill_entries.append(backfill)
+                    if status is None or entry["status"].lower() == status.lower():
+                        excluded_dates = []
+                        if "excluded_dates" in entry:
+                            excluded_dates = entry["excluded_dates"]
+
+                        backfill = cls(
+                            entry_date=entry_date,
+                            start_date=entry["start_date"],
+                            end_date=entry["end_date"],
+                            excluded_dates=excluded_dates,
+                            reason=entry["reason"],
+                            watchers=entry["watchers"],
+                            status=BackfillStatus[entry["status"].upper()],
+                        )
+
+                        backfill_entries.append(backfill)
 
             except yaml.YAMLError as e:
```

Comment:
What's the point of this exception handling? We catch and immediately throw it?

Reply:
I have adjusted the error handling to print the error and exit.